README.md

```diff
@@ -22,9 +22,45 @@ Then access http://localhost:8080/
 
 ## Contributers
 
-### 2020-21
+### FA2020
 
-- **Hahnbee Lee** - Frontend Developer
+- **Hahnbee Lee** - Developer
+- **Theresa Cho** - Developer
+- **Rose Zhou** - Developer
+- **Sam Zhou** - Developer
+- **Ben Shen** - Developer
+- **Ein Chang** - Developer/Designer
+- **Yuxuan Chen** - Designer
+- **Will Spencer** - TPM
+- **Han Wang** - PM
+
+### SP20
+
+- **Theresa Cho** - Developer
+- **Rose Zhou** - Developer
+- **Ayesha Gagguturi** - Developer
+- **Sam Zhou** - Developer
+- **Ein Chang** - Developer/Designer
+- **Emily Chan** - Designer
+- **Yuxuan Chen** - Designer
+- **Shane Yun** - Designer
+- **Chris Lee** - Designer
+- **Kathleen Xu** - PMM
+- **Will Spencer** - TPM
+- **Han Wang** - PM
+
+### FA19
```

Comment:
Lol can you add me :)

Reply:
That's what you get for not being memorable

Reply:
Looool sorry man 🥺